File: api/src/main/java/com/jakedelivery/api/token/ifs/TokenHelperIfs.java
package com.jakedelivery.api.token.ifs;

import com.jakedelivery.api.token.model.Token;

import java.util.Map;

public interface TokenHelperIfs {
    Token issueAccessToken(Map<String, Object> data);
    Token issueRefreshToken(Map<String, Object> data);
    Map<String, Object> validationTokenWithThrow(String token);
}

File: account/src/main/kotlin/com/jakedelivery/account/token/ifs/TokenHelperIfs.kt
package com.jakedelivery.account.token.ifs

import com.jakedelivery.account.token.model.Token

interface TokenHelperIfs {
    fun issueAccessToken(data: Map<String, Any>?): Token?
    fun issueRefreshToken(data: Map<String, Any>?): Token?
    fun validationTokenWithThrow(token: String?): Map<String, Any>?
}